internal/fleetdb/fleetdb.go

```diff
@@ -31,86 +30,44 @@ func NewFleetDBClient(cfg *app.Configuration) (Client, error) {
 		client.SetToken(cfg.FleetDBToken)
 	}
 
+	slugs := make(map[string]*fleetdb.ServerComponentType)
```

Comment:
I don't understand what this is for?

Reply:
I was trying to list all slugs from fleetdb. It can be changed to `map[string]bool`.

I will update this part once https://github.com/bmc-toolbox/common/pull/14/files is merged

Reply:
That PR got merged so we can update the `bmc-toolbox` dependency.